zipkin-web/src/main/scala/com/twitter/zipkin/web/Handlers.scala

```diff
@@ -383,4 +385,11 @@ class Handlers(mustacheGenerator: ZipkinMustache, queryExtractor: QueryExtractor
           .map(renderTrace(_))
       } getOrElse NotFound
     }
+
+  def handleServices(client: HttpClient, webServiceNameCacheTimeoutInSec: Int): Service[Request, Renderer] =
```

Comment:
plus.. you can actually test it there. ex. in `ZipkinQueryServerFeatureTest` you can look for the default cache control header.